Tests/Unit/HeaderTests.swift

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FirebaseCore
+@testable import FirebaseDataConnect
+import Foundation
+import GRPC
+
+import XCTest
+
+final class HeaderTests: XCTestCase {
+  static var defaultApp: FirebaseApp?
```

Comment:
I know I started this, but at some point we may want to put the defaultApp setup in an common test base class for all the unit tests.

Reply:
Yes, I plan to. Will do it in one of my later PRs.